src/profile/main/tag/Google AdWords Remarketing - Prod_QA-UID26.js

```diff
@@ -66,11 +67,25 @@ try {
                             u.data.params[prefix] = u.data.params[prefix] || {};
                             u.data.params[prefix][e[f].substr(prefix.length + 1)] = b[d];
                         } else if (e[f].indexOf("custom.") == 0) {
+                            if (e[f].substr(7) === "isFlightConfirmation") {
+                                isFlightConfirmation = b[d];
+                            } else if (e[f].substr(7) === "bookingWindow") {
+                                bookingWindow = parseInt(b[d]);
+                            }
```

Comment:
so the idea here is that you want to be passing 3 pieces of data to Google if we're on flight confirmation?
1) isFlightConfirmation
2) the booking window
3) flightBookersMembershipTTL

Do you need all three or would just the third one suffice?